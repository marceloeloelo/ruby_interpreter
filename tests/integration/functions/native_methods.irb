puts()

nil
puts("hola\nchau")
hola
chau
nil
puts('hola\nchau')
hola\nchau
nil
puts('hola\n\nchau')
hola\n\nchau
nil
puts('hola\nchau\n')
hola\nchau\n
nil
puts('hola\nchau\n\n')
hola\nchau\n\n
nil
puts('\nchau\n\n')
\nchau\n\n
nil
puts('\n\n')
\n\n
nil
puts("hola\n\nchau")
hola

chau
nil
puts("hola\nchau\n")
hola
chau
nil
puts("hola\nchau\n\n")
hola
chau

nil
puts("\nchau\n\n")

chau

nil
puts("\n\n")


nil
