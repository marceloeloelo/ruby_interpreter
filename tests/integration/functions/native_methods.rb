puts()
puts("hola\nchau")
puts('hola\nchau')

