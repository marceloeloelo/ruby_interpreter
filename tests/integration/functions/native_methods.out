
hola
chau
hola\nchau
hola\n\nchau
hola\nchau\n
hola\nchau\n\n
\nchau\n\n
\n\n
hola

chau
hola
chau

hola
chau



chau





